src/shared/read-agents.c

```diff
@@ -1372,3 +1295,31 @@ time_t scantime_fim (const char *agent_id, const char *scan) {
     return (ts);
 }
 #endif
+
+/**
+ * @brief Extracts a field from a cJSON array and checks it
+ *       
+ * @param root The cJSON root pointer.
+ * @param field Name of the field to extract.
+ * @retval cJSON* On success.
+ * @retval NULL On error.
+ */
+cJSON *get_json_field(cJSON *root, char *field){
```

Comment:
Now **json_p.h** has two new methods instead of `get_json_field()`: `json_get_string_field()` and `json_get_int_field()`.
These functions only allow a cleaner and more compact code.